zadanie_bazy.sql

```diff
@@ -87,18 +79,20 @@ VALUES
     ('starszy grafik', 'tworzy szate wydania', 12400);
     
 INSERT INTO pracownik 
-	(imie, nazwisko,nazwa_stanowiska, opis_stanowiska, wyplata, adres_ulica, adres_kod, adres_miejscowosc)
+	(imie, nazwisko, adres_id, stanowisko_id)
 VALUES
-	('Janek', 'Kozioł', 1, 2, 3, 4, 5, 6),
-    ('Marta', 'Rycerz', 2, 3, 4, 5, 6, 1),
-    ('Robert', 'Zawada', 3, 4, 5, 6, 1, 2),
-    ('Jolanta', 'Topacz', 4, 5, 6, 1, 2, 3),
-    ('Krzysztof', 'Woda', 5, 6, 1, 2, 3, 4),
-    ('Józek', 'Rączka', 6, 1, 2, 3, 4, 5),
-    ('Marcin', 'Szopa', 3, 4, 7, 8, 3, 4),
-    ('Kinga', 'Głowala', 5, 6, 5, 3, 2, 7);
+	('Janek', 'Kozioł', 1, 2),
+    ('Marta', 'Rycerz', 2, 3),
+    ('Robert', 'Zawada', 3, 4),
+    ('Jolanta', 'Topacz', 4, 5),
+    ('Krzysztof', 'Woda', 5, 6),
+    ('Józek', 'Rączka', 6, 1),
+    ('Marcin', 'Szopa', 3, 4),
+    ('Kinga', 'Głowala', 5, 6);
     
 -- 13. Pobiera pełne informacje o pracowniku (imię, nazwisko, adres, stanowisko)
-SELECT p.imie, p.nazwisko, p.adres_ulica, p.adres_kod, p.adres_miejscowosc, p.nazwa_stanowiska FROM pracownik AS p;
+SELECT p.imie, p.nazwisko, p.adres_id, p.stanowisko_id FROM pracownik AS p, stanowisko AS s WHERE p.stanowisko_id = s.id;
 -- 14. Oblicza sumę wypłat dla wszystkich pracownikow w firmie
-SELECT SUM(wyplata) AS sum_wyplata FROM pracownik 
\ No newline at end of file
+SELECT SUM(wyplata) AS sum_wyplata FROM stanowisko;
```

Comment:
(-) To zapytanie zwróci sumę wypłat dla wszystkich stanowisk, a nie dla pracowników. Co jak będziemy mieli przykładowo tylko 1 starszego grafika w firmie?